specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md

```diff
@@ -0,0 +1,247 @@
+# AppLifecycle - State Notifications
+
+Reunion introduces a new AppLifecycle component that provides a core set of functionality related to app activation and lifecycle. Much of this functionality is directly equivalent - or very similar - to existing functionality within the platform. However, there is value in abstracting this functionality so that all app types can use it, delivering it out-of-band from the OS, and making it available as open-source. The totality of AppLifecycle features in v1 is as follows (additional features may be added in later releases):
```

Comment:
General note - please wrap your markdown at 80 characters. VSCode and others have extensions that will do this for you, like Prettier and Markdown-all-in-one.

Reply:
Please clarify: you want a hard return at col 80? Also, I'm not seeing anything in Prettier or Markdown-all-in-one that do this.